src/libostree/ostree-sysroot-deploy.c

```diff
@@ -1211,10 +1211,6 @@ get_kernel_from_tree_usrlib_modules (OstreeSysroot *sysroot, int deployment_dfd,
     }
   glnx_close_fd (&fd);
 
-  /* look for a aboot.cfg file. */
-  if (!ot_openat_ignore_enoent (ret_layout->boot_dfd, "aboot.cfg", &fd, error))
```

Comment:
I think this code was worse than dead - it always leaked a file descriptor if we found devicetree.

See line 1225 below.

Reply:
yes, but there are no such files in practice, so you will always get ENOENT here.